packages/openactive-integration-tests/ARCHITECTURE.md

```diff
@@ -83,7 +83,7 @@ Then use it within the describe blocks as so:
 
 ```jsx
 (new GetMatch({
-  state, flow, logger, configuration,
+  state, flow, logger,
```

Comment:
This field is unused by the `GetMatch` constructor

Reply:
Yep, can be removed 👍